.azure-pipelines/apiview.yml

```diff
@@ -10,6 +10,7 @@ trigger:
       - .azure-pipelines/apiview.yml
       - src/dotnet/APIView
       - src/java/apiview-java-processor
+      - src/swift
```

Comment:
This is not going to make any change since we don't run swift parser inside APIView and developer run this tool manually to create review and upload the file.